lib/vagrant-libvirt/util/mod_impact_flags.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# Ripped from https://libvirt.org/html/libvirt-libvirt-domain.html#virDomainModificationImpact
+module VagrantPlugins
+  module ProviderLibvirt
+    module Util
+      module ModImpactFlags
+        VIR_DOMAIN_AFFECT_CURRENT = 0 # affect current domain state
+        VIR_DOMAIN_AFFECT_LIVE = 1 # affect running domain state
+        VIR_DOMAIN_AFFECT_CONFIG = 2 # affect persistent domain state
```

Comment:
Can you keep these with the rest of domain_flags.rb

Reply:
Yes, let's do that.